packages/augur-core/source/contracts/reporting/RepPriceOracle.sol

```diff
@@ -1,25 +1,35 @@
-pragma solidity 0.5.10;
+pragma solidity 0.5.15;
 
 import 'ROOT/IAugur.sol';
 import 'ROOT/libraries/Initializable.sol';
 import 'ROOT/uniswap/interfaces/IUniswapV2.sol';
 import 'ROOT/uniswap/interfaces/IUniswapV2Factory.sol';
 import 'ROOT/reporting/IRepPriceOracle.sol';
+import 'ROOT/ens/IENSRegistry.sol';
+import 'ROOT/ens/IENSResolver.sol';
 
 
 contract RepPriceOracle is IRepPriceOracle, Initializable {
 
+    uint256 constant Q112 = 2**112;
+
+    bytes32 public constant UNISWAP_REGISTRY_ENS_NAME = 0xf259e1e59b6e1e9de21a18289a53332ebc255d2e4ece8a91e0e90c628a9c7f87; // "uniswapv2.eth"; // TODO: set this to the actual namehash of the owned ENS name
```

Comment:
Use a contract address, not an ENS name.  ENS ownership is (usually) centralized/controlled so the ENS name owner for `uniswapv2.eth` could change what it points to at any time.

Reply:
The point is is to allow it to change once after our initial deployment based on the uniswap team setting the ENS name